src/training/scheduler.h

```diff
@@ -146,9 +150,14 @@ class Scheduler : public TrainingObserver {
       : options_(options), state_(state) {
     ABORT_IF(state_->factor != 1, "state.factor unexpectedly not 1 at this point??");
     updateLearningRate(*state);
+    installSignalHandlers_();
```

Comment:
This seems like an odd place to put that and rather not well motived. @snukky can you suggest a better location?

Reply:
It's in the constructor, so should be OK.

Reply:
Actually, ignore that comment. GitHub cut off the function name so weirdly that I thought it was inside getScheduledLRDecayFactor().

Reply:
The constructor is fine.